CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - feat(maven): Add maven target to deploy to Maven Central (#258)
 - feat(symbol-collector): Add symbol-collector target (#266)
 - ref(maven): Support BOM files in `maven` target (#270)
+- ref(maven): Optional `android` config (#271)
```

Comment:
These are both new features, not refactors as refactors should not change existing behavior (thus, not documented in changelog 😄 )

```suggestion
- feat(maven): Support BOM files in `maven` target (#270)
- feat(maven): Optional `android` config (#271)
```